problem_builder/tests/unit/test_problem_builder.py

```diff
@@ -1,9 +1,15 @@
-import unittest
 import ddt
+import unittest
+
 from mock import MagicMock, Mock, patch
+from random import random
+
 from xblock.field_data import DictFieldData
+
 from problem_builder.mcq import MCQBlock
-from problem_builder.mentoring import MentoringBlock, MentoringMessageBlock, _default_theme_config
+from problem_builder.mentoring import (
+    MentoringBlock, MentoringMessageBlock, _default_theme_config, _default_options_config
```

Comment:
@bradenmacdonald #103 removes `_default_theme_config` completely.

The `pylint` command we run on CircleCI currently [enables](https://github.com/open-craft/problem-builder/blob/master/circle.yml#L18) only a handful of checks. `unused-import` is not one of them, that's why `_default_theme_config` fell through the cracks here.